sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/csvExpressions.scala

```diff
@@ -93,8 +98,6 @@ case class CsvToStructs(
     }
   }
 
-  val nameOfCorruptRecord = SQLConf.get.getConf(SQLConf.COLUMN_NAME_OF_CORRUPT_RECORD)
```

Comment:
Reading of the config can be moved into `parser` initialization. In that case, it will not influence on planning.